jsapp/scss/components/_kobo.map.scss

```diff
@@ -400,6 +400,27 @@ body .leaflet-control-layers,
   }
 }
 
+.map-settings__querylimit {
+  label {
```

Comment:
All these styles for `.map-settings__querylimit label` and `.map-settings__querylimit label input` are not used, please delete them